### ISSUE TITLE
cosmetics: automatically adjust the items of a list so that they are fully presented and not cut off

### ISSUE THREAD

**Issue by Jefgeno** [protofire/omen-exchange#944]
![paste](https://user-images.githubusercontent.com/62712672/87674111-d2cd6a80-c775-11ea-821f-8fe8a3396446.png)

**Comment by KadenZipfel** [comment]
Can you elaborate on what you mean by this?

**Comment by Jefgeno** [comment]
Yes, I suggest that while scrolling the full market information are always visible. In my screenshot the half of the first market is cut away, so you can ony see the outcomes of the market.
